### ISSUE TITLE
New fonts and new parameters

### ISSUE THREAD

**Issue by miaoski** [nicholastoddsmith/pythonml#8]
Hi.  Thank you for DeepOCR.  It looks quite useful.
I wanted to train a new model with Microsoft's ARIALUNI.TTF.  I changed nothing but `consola.ttf` to `ARIALUNI.TTF`, `rm -fr TFModel Tst Trn Tst.csv Trn.csv` then `mkdir TFModel Tst Trn`.
As I ran `DeepOCR.py somefile.gif`, it began the training process and gave me this error.

```
Caused by op 'ocrnet0/softmax_cross_entropy_with_logits_sg', defined at:
  File "DeepOCR.py", line 132, in <module>
    CNN = LoadNets()                    #CNNs
  File "DeepOCR.py", line 117, in LoadNets
    CNN = [MakeNet('ocrnet' + str(i)) for i in range(NNET)]
  File "DeepOCR.py", line 117, in <listcomp>
    CNN = [MakeNet('ocrnet' + str(i)) for i in range(NNET)]
  File "DeepOCR.py", line 40, in MakeNet
    return ANNC(IS, ws, batchSize = 512, learnRate = 2e-5, maxIter = 64, name = nn, reg = 1e-5, tol = 1e-2, verbose = True, X = TFIM, Y = TFYH)
  File "/home/charlie/.local/lib/python3.6/site-packages/TFANN.py", line 471, in __init__
    super().__init__(_IS, _NA, batchSize, learnRate, loss, maxIter, name, optmzrn, pad, reg, tol, verbose, X, Y)
  File "/home/charlie/.local/lib/python3.6/site-packages/TFANN.py", line 213, in __init__
    self.CreateModel()                                      #Populate TensorFlow graph
  File "/home/charlie/.local/lib/python3.6/site-packages/TFANN.py", line 226, in CreateModel
    self.loss = tf.reduce_mean(self.LF(self.O[-1], self.Y))    #Loss term
  File "/home/charlie/.local/lib/python3.6/site-packages/TFANN.py", line 143, in <lambda>
    'smce': lambda YH, Y : tf.nn.softmax_cross_entropy_with_logits(labels = Y, logits = YH)}.get(name)
  File "/home/charlie/tensorflow/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py", line 250, in new_func
    return func(*args, **kwargs)
  File "/home/charlie/tensorflow/lib/python3.6/site-packages/tensorflow/python/ops/nn_ops.py", line 1960, in softmax_cross_entropy_with_logits
    labels=labels, logits=logits, dim=dim, name=name)
  File "/home/charlie/tensorflow/lib/python3.6/site-packages/tensorflow/python/ops/nn_ops.py", line 1875, in softmax_cross_entropy_with_logits_v2
    precise_logits, labels, name=name)
  File "/home/charlie/tensorflow/lib/python3.6/site-packages/tensorflow/python/ops/gen_nn_ops.py", line 4575, in _softmax_cross_entropy_with_logits
    name=name)
  File "/home/charlie/tensorflow/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/home/charlie/tensorflow/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/home/charlie/tensorflow/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

InvalidArgumentError (see above for traceback): logits and labels must be same size: logits_size=[111616,63] labels_size=[32768,63]
	 [[Node: ocrnet0/softmax_cross_entropy_with_logits_sg = SoftmaxCrossEntropyWithLogits[T=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:GPU:0"](ocrnet0/softmax_cross_entropy_with_logits_sg/Reshape, ocrnet0/softmax_cross_entropy_with_logits_sg/Reshape_1)]]
```

I use Ubuntu 16.04, CUDA 9, libCUDNN 7 on nVidia driver 390.30.  Tensorflow was installed by `pip3 install tensorflow-gpu`.  Any hints?  Thank you!

**Comment by miaoski** [comment]
P.S. Since padding is done with ` ` (space), I can't train a model with all lowercases without space. :)  That's not serious, though.

**Comment by nicholastoddsmith** [comment]
Looks like you did not set the image size appropriately. Use the image sizes of the generated training images as a guide. _IS_ must match between the two files. Closing as this is an issue resulting from your modifications.
